dynamic_config/BUILD

```diff
@@ -0,0 +1,54 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
```

Comment:
Can we also add a top level `README.md` indicating what this directory is for?

Additionally would you mind moving it under `test/integration` where most of test utilities and fixtures live? Making this a top level directory doesn't seem to be the right move, since it doesn't have anything to do with Nighthawk itself, just with the test suite built around it.

Reply:
Added a readme.

I don't think `test/integration` is the right place since the components can be used outside of those fixtures e.g. similar to how you can run nighthawk outside of these fixtures, you can use these components for an Envoy there without any of the integration test stuff.

Reply:
That is fair, if you think this belongs to the top level directory, can we add a README.md outlining what this is and how can one start and use the component?

Reply:
My bad, just noticed that the README is already present. Could you expand it with an example of use though?

Ok to go with a TODO and postpone to a later PR if you prefer.

Reply:
added a TODO here.